package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "combine-errors",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
Can you revert the version change? Most people have a release step that will do this, create tags, etc.